acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -69,23 +143,19 @@ def rpc_blocking_request(broker, targets,
   # Event machine is required by the ruby-pcp-client gem
   # https://github.com/puppetlabs/ruby-pcp-client
   em = Thread.new { EM.run }
+  Thread.pass until EventMachine.reactor_running?
```

Comment:
`EM` is an alias to `EventMachine` so this code works, but for consistency make this EM.reactor_running? or change the preceding line to be `EventMachine.run`